source/docs/one-command-deployment.md

```diff
@@ -222,10 +222,10 @@ $ npm i -g now
 
 2. Add a build script to your `package.json` file:
 
-```
+```json
 {
   "scripts": {
-    "build: "hexo deploy"
+    "build: "hexo generate"
```

Comment:
It doesn't seems normal.

Also, the deploy button should be added to all the languages.

Reply:
Hi @SukkaW 

Looks like there was a typo in the json that I fixed.

What do you mean "all the languages"? Are there other files that mention ZEIT Now?